bokehjs/src/less/tabs.less

```diff
@@ -1,5 +1,25 @@
 @import "./_mixins.less";
 
+:host {
+  // The border opposite the divider
+  --block-start-border-width: 3px;
+
+  --close-icon-hover-background-color: var(--error);
+  // Should these dimensions be px or rem?
```

Comment:
For now we should keep pixels, but eventually replace them with relative units. Can you start an issue for this?